### PR TITLE
Inputsize invariant rsgcn

### PR DESCRIPTION
**Pull request by corochann**
This is a test to explicitly check that model forward result is same even if input size is bigger (with atom & adjacency matrix filled with 0).

### CODE REVIEW

File: chainer_chemistry/utils/extend.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+def extend_adj(adj, out_size, axis=None, value=0):
+    axis = axis or [-1, -2]
+    return extend_arrays_to_size(
+        adj, out_size=out_size, axis=axis, value=value)
+
+
+def extend_arrays_to_size(arrays, out_size, axis=-1, value=0):
+    """Extend size of `node` array
```
Comment (mottodora) [inline chainer_chemistry/utils/extend.py]: Do you think users will use `extend_arrays_to_size` function? Anyway, this docstring  is for `extend_node`.

File: chainer_chemistry/utils/extend.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        a = [a]
+    return a
+
+
+def extend_node(node, out_size, axis=-1, value=0):
+    return extend_arrays_to_size(
+        node, out_size=out_size, axis=axis, value=value)
+
+
+def extend_adj(adj, out_size, axis=None, value=0):
```
Comment (mottodora) [inline chainer_chemistry/utils/extend.py]: Please add docstring.

File: chainer_chemistry/utils/extend.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+def _to_list(a):
+    if isinstance(a, Iterable):
+        a = list(a)
+    else:
+        a = [a]
+    return a
+
+
+def extend_node(node, out_size, axis=-1, value=0):
```
Comment (mottodora) [inline chainer_chemistry/utils/extend.py]: Please add docstring.

File: chainer_chemistry/utils/extend.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    Returns (numpy.ndarray): extended `node` array, extended place is filled
+        with `value`
+
+    """
+    batch_size = len(arrays)
+
+    in_shape = _to_list(arrays[0].shape)
+    out_shape = [batch_size] + in_shape
+
+    axis = _to_list(axis)
```
Comment (mottodora) [inline chainer_chemistry/utils/extend.py]: I feel if axis contains `0`, raising warning is kind  to user. But it is not necessary to solve.

File: chainer_chemistry/utils/extend.py (hunk 1 above)
Comment (corochann) [reply chainer_chemistry/utils/extend.py]: maybe yes, I update docstring.

File: chainer_chemistry/utils/extend.py (hunk 4 above)
Comment (corochann) [reply chainer_chemistry/utils/extend.py]: ok

**Pull request merged**